domain_config_ui/domain_config_ui.services.yml

```diff
@@ -0,0 +1,16 @@
+services:
+  domain_config_ui.manager:
+    class: Drupal\domain_config_ui\DomainConfigUIManager
+    arguments: ['@config.storage', '@domain.loader', '@language_manager']
```

Comment:
On second thoughts, I am swapping all allowed config with the manager so it might not be helping much.

I can change back to extending the negotiator as before and see what problems I encounter.